generative_data_prep/utils/utils.py

```diff
@@ -142,6 +142,23 @@ def data_prep_arg_builder(parser: argparse.ArgumentParser):
         help=
         "keyword used in input json to specify completion, defaults to 'completion"
     )
+
+    parser.add_argument(
+        "--prompter_keyword",
```

Comment:
Should we make these more general like prompt_prefix? I think that naming convention may be more clear

Reply:
It is also too close of a name to prompt_keyword, that would be very confusing

Reply:
I think making the arguments more general is a good idea:
`prompt_prefix` and `prompt_suffix` makes more sense to me.